apps/rush-lib/src/cli/scriptActions/BulkScriptAction.ts

```diff
@@ -118,7 +124,7 @@ export class BulkScriptAction extends BaseScriptAction {
       isIncrementalBuildAllowed: this._isIncrementalBuildAllowed,
       ignoreMissingScript: this._ignoreMissingScript,
       ignoreDependencyOrder: this._ignoreDependencyOrder,
-      packageDepsFilename: Utilities.getPackageDepsFilenameForCommand(this._commandToRun)
+      packageDepsFilename: Utilities.getPackageDepsFilenameForCommand(this._commandToRun[0])
```

Comment:
Ideally this should use the name of the command that actually executed, in case there is a collision between multiple Rush commands.